python/tests/test_session.py

```diff
@@ -121,3 +121,28 @@ def test_it_quits_the_driver(self):
         session.start()
 
         session.stop()
+
+
+class TestUpdateResult(object):
+
+    def test_passing_case(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)